aidatlu/main/tlu.py

```diff
@@ -489,12 +489,14 @@ def run(self) -> None:
 
         if save_data:
             self.h5_file.close()
-        if interpret_data_bool:
             interpret_data(self.raw_data_path, self.interpreted_data_path)
+
         self.log.success("Run finished")
 
 
 if __name__ == "__main__":
+    import uhal
```

Comment:
Generally speaking, this is discouraged, although I see and agree with the reason to import it here. Let me think if there is a nicer way.